defaults/main.yml

```
@@ -7,3 +7,5 @@ coredns_config_file: ""
 
 # Use a local binary instead of the official release.
 coredns_binary_local_dir: ""
+
+coredns_zone_files_path: "coredns/zones"
```

You need to put the end of file newline back to get the tests to pass.

Let's include the whole fileglob here to make this more extensible.

```suggestion
coredns_zone_files_path:
- "coredns/zones/*"
```